.gitlab-ci.yml

```diff
@@ -365,6 +365,8 @@ build_win:
   before_script: []
   script:
     - call .gitlab-scripts/setup_artifactory.cmd
+    - set "PATH=C:\tools\msys64\mingw64\bin;%PATH%"
+    - set "PATH=%PROGRAMFILES(X86)%\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin;%PATH%"
```

Comment:
This path comes after 
%VCINSTALLDIR%\Common7\Tools\VsDevCmd.bat

VsDevCmd.bat sets the Visual Studio related environment variables that are necessary for compiling, linking and building applications on the command line.

If you have to set this here... again